frame/system/src/lib.rs

```diff
@@ -146,7 +147,7 @@ pub trait Trait: 'static + Eq + Clone {
 		+ Clone;
 
 	/// The aggregated `Call` type.
-	type Call: Debug;
+	type Call: Debug + Dispatchable<Origin = Self::Origin>;
```

Comment:
Related to my other comment: Why Call has to be bounded here?

Reply:
It is no longer bounded.